server/src/validators/auth.js

```diff
@@ -0,0 +1,24 @@
+const { check, validationResult } = require("express-validator");
+
+exports.validateSignupRequest = [
+    check('firstName')
+    .notEmpty()
+    .withMessage('firstName is required'),
+    check('lastName')
+    .notEmpty()
+    .withMessage('lastName is required'),
+    check('email')
+    .isEmail()
+    .withMessage('lastName is required'),
+    check('password')
+    .isLength({min: 6})
+    .withMessage('Password must be off 6 character long')
+];
+
+exports.isRequestValidated = (req, res, next) => {
+    const errors = validationResult(req);
+    if(errors.array().length > 0){
+        return res.status(400).json({ error: errors.array()[0].msg})
+    }
+    next();
+}
```

Comment:
For the last time, we are not exporting in this manner use `module.export = .....` in this last

Reply:
my mistake

Reply:
Export thing pls